Database/getting-started-with-MySQL-DBaaS.md

```diff
@@ -0,0 +1,49 @@
+{{{
+  "title": "Getting Started with MySQL DBaaS",
+  "date": "08-14-2015",
+  "author": "Christine Parr",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": true
```

Comment:
I just copied and pasted that header from another KB - I think AppFog - so it can be changed if needed.